README.md

```diff
@@ -1,2 +1,136 @@
-# Leaderboard
- 
+<a name="readme-top"></a>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠️ Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard:Setup Project ] <a name="about-project"></a>
+
+** Leaderboard** is a simple website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
+
+
+## 🛠️ Built With <a name="built-with">
+### Tech Stack <a name="tech-stack"></a>
+- HTML
+- JAVASCRIPT
+
+</a>
+
+
+### Key Features <a name="key-features"></a>
+
+- **[Good and Simple Design]**
+ **[UX/UI accessibility ]**
+ **[Responsive layout]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- browser like google chrome
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- clone the project using by running the following command :
+
+`git clone`
+`https://github.com/hagan199/Leaderboard.git`
+
+###  Install
+
+ ```install
+ npm install
+ ```
```

Comment:
- [ ] I kindly suggest that you add the step for usage of your web app. for example
[OPTIONAL]
```
### Usage

To run the project, execute the following command:


- Start the development server
`   ```sh
`   npm start
`   ```
`
```

Reply:
Noted